src/framework/uicomponents/view/textinputfieldmodel.cpp

```diff
@@ -80,6 +80,7 @@ void TextInputFieldModel::loadShortcuts()
 {
     //! NOTE: from navigation actions
     static std::vector<std::string> actionCodes {
+        "apply-current-palette-element",
```

Comment:
Best place I could find to allow for this command to be triggered from a TextInputField. Otherwise, nothing would happen while palette searching and using this command...